packages/opentelemetry-exporter-collector/src/types.ts

```diff
@@ -339,6 +339,7 @@ export interface CollectorExporterConfigBase {
   serviceName?: string;
   attributes?: Attributes;
   url?: string;
+  connectionReuse?: boolean;
```

Comment:
What do you think about just calling this `keepAlive`?

Reply:
Yes, this shouldn't be a problem.